lab-teddy/app/scss/base.scss

```diff
@@ -0,0 +1,43 @@
+// **** VARS **** //
+$brand-primary: #BADDAD;
+$brand-secondary: #AFAFAF;
+
+$padding: .5em;
+
+$font-size-primary: 1.025em;
+
+// **** MIXINS **** //
+@mixin bubble{
+  border-radius: .25em;
+  font-family: sans-serif;
+}
+
+@mixin drop{
```

Comment:
Great work with this mixin and including a previous mixin. Oh the power of scss.